grails-app/controllers/au/org/ala/merit/ProjectController.groovy

```diff
@@ -225,7 +225,8 @@ class ProjectController {
             model.details.visible = model.details.visible && userHasViewAccess
 
             boolean reportsVisible = config.includesContent(ProgramConfig.ProjectContent.REPORTING) && userHasViewAccess
-            Map reportingTab = [label: 'Reporting', visible:reportsVisible, type:'tab', template:'projectReporting', reports:project.reports, stopBinding:true, services: config.services, scores:scores, hideDueDate:true, isAdmin:user?.isAdmin, isGrantManager:user?.isCaseManager]
+
+            Map reportingTab = [label: 'Reporting', visible:reportsVisible, type:'tab', template:'projectReporting', reports:project.reports, stopBinding:true, services: config.services, scores:scores, hideDueDate:true, isAdmin:user?.isAdmin, isGrantManager:user?.isCaseManager, isRdpReport:showMeriPlanComparison, declarationTemplate:config.getDeclarationTemplate()]
```

Comment:
What is the model variable "isRdpReport" used for (I can't see it used in this PR)?  Can it be deleted?